data/language/nl-NL.yml

```diff
@@ -1111,75 +1111,75 @@ strings:
   1083: File contains invalid data
   1084: File is not a single player saved game
   1085: File is not a two player saved game
-  1086: Please wait...
-  1087: Initialising...
-  1088: Loading...
+  1086: Even geduld...
+  1087: Initialiseren...
+  1088: Laden...
   1089: "Installing new data: "
 
-  1090: White
-  1091: Translucent
-  1092: "{COLOUR WINDOW_2}Construction Marker:"
-  1093: "{COLOUR WINDOW_2}Vehicles min. display scale:"
-  1094: "{COLOUR WINDOW_2}Station names min. display scale:"
-  1095: Full-scale
-  1096: Half-scale
-  1097: Quarter-scale
-  1098: Eighth-scale
+  1090: Wit
+  1091: Doorschijnend
+  1092: "{COLOUR WINDOW_2}Bouwmarkering:"
+  1093: "{COLOUR WINDOW_2}Min. zoomniv. voertuigen:"
+  1094: "{COLOUR WINDOW_2}Min. zoomniv. stationsnamen:"
+  1095: Volledige zoom
+  1096: Halve zoom
+  1097: Kwartzoom
+  1098: Achtste zoom
   1099: "{SMALLFONT}{COLOUR BLACK}Select minimum view scale to show vehicles at"
   1100: "{SMALLFONT}{COLOUR BLACK}Select minimum view scale to show station names on"
 
-  1101: "{COLOUR WINDOW_2}Main colour scheme:"
-  1102: "{COLOUR BLACK}Steam Locomotives:"
-  1103: "{COLOUR BLACK}Diesel Locomotives:"
-  1104: "{COLOUR BLACK}Electric Locomotives:"
-  1105: "{COLOUR BLACK}Multiple Units:"
-  1106: "{COLOUR BLACK}Passenger Vehicles:"
-  1107: "{COLOUR BLACK}Freight Vehicles:"
-  1108: "{COLOUR BLACK}Buses:"
-  1109: "{COLOUR BLACK}Trucks:"
-  1110: "{COLOUR BLACK}Aircraft:"
-  1111: "{COLOUR BLACK}Ships:"
-  1112: "{SMALLFONT}{COLOUR BLACK}Company headquarters and details"
-  1113: "{SMALLFONT}{COLOUR BLACK}Company owner and status"
-  1114: "{SMALLFONT}{COLOUR BLACK}Company finances"
+  1101: "{COLOUR WINDOW_2}Hoofdkleurstelling:"
+  1102: "{COLOUR BLACK}Stoomlocomotieven:"
+  1103: "{COLOUR BLACK}Diesellocomotieven:"
+  1104: "{COLOUR BLACK}Electrische locomotieven:"
+  1105: "{COLOUR BLACK}Treinstellen:"
+  1106: "{COLOUR BLACK}Passagiersvoertuigen:"
+  1107: "{COLOUR BLACK}Goedereenvoertuigen:"
+  1108: "{COLOUR BLACK}Bussen:"
+  1109: "{COLOUR BLACK}Vrachtwagens:"
+  1110: "{COLOUR BLACK}Vliegtuigen:"
+  1111: "{COLOUR BLACK}Schepen:"
+  1112: "{SMALLFONT}{COLOUR BLACK}Bedrijfshoofdkwartier en -details"
+  1113: "{SMALLFONT}{COLOUR BLACK}Bedrijfseigenaar en -status"
+  1114: "{SMALLFONT}{COLOUR BLACK}Bedrijfsfinanciën"
   1115: "{SMALLFONT}{COLOUR BLACK}Cargo delivered"
-  1116: "{SMALLFONT}{COLOUR BLACK}Company colour scheme"
+  1116: "{SMALLFONT}{COLOUR BLACK}Kleurstelling bedrijf"
   1117: "{SMALLFONT}{COLOUR BLACK}Company challenge for this game"
-  1118: "{COLOUR WINDOW_2}Special colour schemes used for:"
-  1119: "{SMALLFONT}{COLOUR BLACK}Select main colour"
-  1120: "{SMALLFONT}{COLOUR BLACK}Select secondary colour (used only on some vehicle types)"
+  1118: "{COLOUR WINDOW_2}Gebruik speciale kleurstellingen voor:"
+  1119: "{SMALLFONT}{COLOUR BLACK}Selecteer hoofdkleur"
+  1120: "{SMALLFONT}{COLOUR BLACK}Selecteer secondaire kleur (wordt niet op alle voertuigen gebruikt)"
```

Comment:
Secundair lijkt gebruikelijker te zijn https://www.schrijfwijzer.nl/verwarwoordenboek/verwarwoord/473/secondair-secundair https://taal.vrt.be/secundair-secondair